packages/now-cli/src/util/dev/builder-cache.ts

```diff
@@ -39,10 +38,14 @@ const localBuilders: { [key: string]: BuilderWithPackage } = {
   },
 };
 
-const bundledBuilders = Object.keys(devDependencies).filter(d =>
+const bundledBuilders = Object.keys(pkg.devDependencies).filter(d =>
```

Comment:
Is `bundledBuilders` still used? I think it should be removed.

Reply:
Or perhaps it should be `const bundledBuilders = getBundledBuilders();`

Reply:
I replaced every `bundledBuilders` with `getBundledBuilders()`